lib/models/BucketInfo.js

```diff
@@ -147,14 +155,16 @@ class BucketInfo {
         this._deleted = deleted || false;
         this._serverSideEncryption = serverSideEncryption || null;
         this._versioningConfiguration = versioningConfiguration || null;
-        this._locationConstraint = locationConstraint || null;
+        this._locationConstraint = locationConstraint ?
+            locationConstraint.split(zenkoSeparator)[0] : null;
         this._readLocationConstraint = readLocationConstraint || null;
         this._websiteConfiguration = websiteConfiguration || null;
         this._replicationConfiguration = replicationConfiguration || null;
         this._cors = cors || null;
         this._lifecycleConfiguration = lifecycleConfiguration || null;
         this._uid = uid || uuid();
         this._isNFS = isNFS || null;
+        this._ingestion = ingestion ? { status: 'Enabled' } : null;
```

Comment:
I think this is just fine, and can't think of bucket creation where ingestion is not enabled at the start, but maybe just to be safe, use the `setIngestion` method here to set this instance variable.